core/server/index.js

```diff
@@ -78,8 +79,8 @@ function init(options) {
         debug('Themes & apps done');
 
         models.init();
-        // @TODO: this is temporary, replace migrations with a warning if a DB exists
-        return db.bootUp();
+    }).then(function () {
+        return knexMigrator.isDatabaseOK();
```

Comment:
> Ghost will die and tell you what todo (we forward the error from knex-migrator).

Note: we should probably have a nicely documented exit code here.

> When your database was never initialised, you need to run knex-migrator init before your can start node index.js. So one more manual step if you don't use Ghost-CLI.

Ghost-CLI is going to be *the* way to install Ghost. The only reason you wouldn't use it is if you are developing Ghost locally. In that instance, having a powerful set of tools for managing the DB is, IMO an advantage, not a disadvantage. 

Another way to look at this, is that we've had migrations baked into Ghost since day 1 and IMO it's been a source of pain. There's no understanding that Ghost might do this on startup, or why, and there's no good tooling to help you get unstuck if Ghost does the wrong thing. Therefore, with the introduction of the CLI, the idea was to separate these two things and try it out this way to see if the grass is greener.

If it turns out this is not a better approach, we can always bake it back into Ghost boot again later?